src/Illuminate/View/ViewServiceProvider.php

```diff
@@ -73,8 +73,9 @@ public function registerBladeEngine($resolver)
         // instance to pass into the engine so it can compile the views properly.
         $app->singleton('blade.compiler', function ($app) {
             $cache = $app['config']['view.compiled'];
+            $shouldCache = $app['config']['view.should_cache'] ?: true;
```

Comment:
I think this doesn't work. If I set ``should_cache`` to false it's just going to be true again.